dolphinscheduler-task-plugin/pom.xml

```diff
@@ -51,5 +51,6 @@
         <module>dolphinscheduler-task-emr</module>
         <module>dolphinscheduler-task-blocking</module>
         <module>dolphinscheduler-task-zeppelin</module>
+        <module>dolphinscheduler-task-jupyter</module>
```

Comment:
Add this dependency to module https://github.com/apache/dolphinscheduler/blob/dev/dolphinscheduler-task-plugin/dolphinscheduler-task-all/pom.xml?

Reply:
Looks like already added here? 
![image](https://user-images.githubusercontent.com/34905992/167794968-473ad309-1d66-4591-a628-fc8875b7b003.png)

Reply:
> Looks like already added here? 
> 
> ![image](https://user-images.githubusercontent.com/34905992/167794968-473ad309-1d66-4591-a628-fc8875b7b003.png)
> 
> 

Uh yes I missed that as reviewed only my phone.